### DISCUSSION TITLE
Full Crawl in Microsoft search not recreating index from scratch

### DISCUSSION THREAD

**Discussion by gupta-pa** [microsoftgraph/msgraph-connectors-sdk#28] [Q&A] [answered]
I have been trying Graph Connector SDK to index external content into MS Search. Created the custom connector, published it, ran full crawl and content was successfully indexed (let's say 5 items). Now, for the same connector, I tried to insert another 2 items using Graph Explorer. Total 7 items indexed.

From the admin dashboard, when I ran Full crawl for this connector, I was expecting total 5 indexed items, but it was still 7. Can someone please explain why the whole index is not re-created with 5 items? Is this a known functionality?

Note: Using, graph connector SDK we are not setting Item id. But using Graph explorer, we have to pass a unique id for each item.

**Comment by rchanda1392** [comment, marked as answer]
@gupta-pa the SDK will only be able to delete items indexed using the SDK on it's own. The other 2 items needs to be removed using the same route that they were indexed with.
